app/src/main/java/ai/elimu/vitabu/ui/storybook/WordViewAdapter.kt

```diff
@@ -42,14 +42,10 @@ internal class WordViewAdapter(
                 readingLevelPosition
             )
 
-            if (words[position] != null) {
-                holder.itemView.setOnClickListener { v: View? ->
-                    listener.onItemClick(
-                        words[position], v, position
-                    )
-                }
-            } else {
-                holder.itemView.setOnClickListener(null)
+            holder.itemView.setOnClickListener { v: View? ->
+                listener.onItemClick(
+                    words[position], text = wordsInOriginalText[position] ?: "", v, position
```

Comment:
```suggestion
                    words[position], text = wordsInOriginalText[position] ?: "", view, position
```

Reply:
@jo-elimu This isn't related to the changes of the PR, but anyway I've updated it.